app/views/twoFactorAuth/index.njk

```diff
@@ -0,0 +1,53 @@
+{% extends "../layout.njk" %}
+
+{% block page_title %}
+Sign-in method - GOV.UK Pay
+{% endblock %}
+
+{% block mainContent %}
+<div class="column-full">
+  <a href="{{profile}}" class="link-back">My profile</a>
+</div>
+<div class="column-full">
+  <h1 class="heading-large page-title">
+    Sign-in method
+  </h1>
+</div>
+<div class="column-two-thirds">
+  {% if authenticatorMethod == 'SMS' %}
+    <p>You currently use text message codes to sign in to GOV.UK Pay.</p>
+    <form method="post" action="{{index}}">
+      <input id="csrf" name="csrfToken" type="hidden" value="{{csrf}}"/>
+      <div class="form-group">
+        <button class="button" type="submit">
+          Use an authenticator app instead
+        </button>
+      </div>
+    </form>
+  {% endif %}
+  {% if authenticatorMethod == 'APP' %}
+    <p>You currently use an authenticator app to sign in to GOV.UK Pay.</p>
+    <form method="post" action="{{index}}">
+      <input id="csrf" name="csrfToken" type="hidden" value="{{csrf}}"/>
+      <div class="form-group">
+        <button class="button" type="submit">
+          Set up a different authenticator app
+        </button>
+      </div>
+    </form>
+  {% endif %}
+  <div>
+    <details>
+      <summary>What is an Authenticator app?</summary>
+      <h2 class="heading-small">Authenticator apps</h2>
+      <p>The GOV.UK Pay self-service website requires users to use a security code to sign-in, the code is sent to a users mobile number. It’s possible to get the code from an Authenticator app instead.</p>
+      <p>Users can use any Authenticator app they like but the most common are:</p>
+      <ul class="list list-bullet">
+        <li><a href="https://support.google.com/accounts/answer/1066447">Google Authenticator</a></li>
+        <li><a href="https://authy.com/">Authy</a></li>
+        <li><a href="https://duo.com/">Duo</a></li>
```

Comment:
It makes me sad that none of these links are particularly good. The Google Authenticator one assumes you’re using Google Authenticator just for your Google account (which might confuse people) and the Duo landing page tries to sell to companies rather than users. Even the Authy website (which is the best) basically assumes you already know what an authenticator app is and is built around their guides for high-profile sites.

Other apps we could possibly add to the list include Microsoft Authenticator (again, its site assumes you’re using it to secure a Microsoft Account or an account with an organisation that uses Microsoft’s Azure cloud services) and LastPass Authenticator (fairly good site but assumes you’re probably using LastPass).